h2/src/main/org/h2/expression/function/Function.java

```diff
@@ -469,6 +485,19 @@
         addFunctionWithNull("SIGNAL", SIGNAL, 2, Value.NULL);
         addFunctionNotDeterministic("ESTIMATED_ENVELOPE", ESTIMATED_ENVELOPE, 2, Value.LONG);
         addFunction("H2VERSION", H2VERSION, 0, Value.STRING);
+        
+        // JSON
+        addFunction("JSON_FIELD", JSON_FIELD, 2, Value.JSON);
+        addFunction("JSON_FIELD_TEXT", JSON_FIELD_TEXT, 2, Value.STRING);
+        addFunction("JSON_FIELD_PATH", JSON_FIELD_PATH, 2, Value.JSON);
+        addFunction("JSON_FIELD_PATH_TEXT", JSON_FIELD_PATH_TEXT, 2, Value.STRING);
+        addFunction("JSON_CONTAINS", JSON_CONTAINS, 2, Value.BOOLEAN);
+        addFunction("JSON_EXISTS", JSON_EXISTS, 2, Value.BOOLEAN);
+        addFunction("JSON_EXISTS_ANY", JSON_EXISTS_ANY, 2, Value.BOOLEAN);
+        addFunction("JSON_EXISTS_ALL", JSON_EXISTS_ALL, 2, Value.BOOLEAN);
+        addFunction("JSON_CONCAT", JSON_CONCAT, 2, Value.JSON);
+        addFunction("JSON_DELETE_FIELD", JSON_DELETE_FIELD, 2, Value.JSON);
```

Comment:
Where all these functions came from? Neither SQL Standard nor PostgreSQL documentation have them with an exception for `JSON_EXISTS` that is described in the standard (but I'm not sure that it is compatible with your implementation).

If you need them to support some operations, it's better to use own class similar to `BinaryOperation` instead. Operations can be mapped to functions only when a similar function exists.

Reply:
Yes, now these functions are used for the work of operators. With the development of the functionality, it is planned to add new ones, but I'll try to correlate names with the standard or the PostgreSQL dialect.